### PR TITLE
Deployments on base sepolia for alpha

### PR DESCRIPTION
**Pull request by giuseppecrj**

### CODE REVIEW

File: contracts/makefile [hunk 1]
```diff
@@ -113,6 +113,25 @@ interact-sepolia :;
...
+
+deploy-alpha-sepolia :;
+	@echo "Deploying $(contract)..."
+	@SAVE_DEPLOYMENTS=1 DEPLOYMENT_CONTEXT=alpha forge script scripts/deployments/$(if $(findstring facet,$(type)),facets/)${contract}.s.sol:${contract} --ffi --rpc-url sepolia --private-key ${TESTNET_PRIVATE_KEY} --broadcast --verify --legacy -vvvv
+
+deploy-alpha-base-sepolia :;
+	@echo "Deploying $(contract)..."
+	@SAVE_DEPLOYMENTS=1 DEPLOYMENT_CONTEXT=alpha forge script scripts/deployments/$(if $(findstring facet,$(type)),facets/)${contract}.s.sol:${contract} --ffi --rpc-url base_sepolia --private-key ${TESTNET_PRIVATE_KEY} --broadcast --verify --verifier blockscout --verifier-url ${BLOCKSCOUT_SEPOLIA_URL}
+
+deploy-alpha-river-testnet :;
```
Comment (jterzis) [inline contracts/makefile:130]: Are you planning on including alpha riverRegistry deployment addresses under packages/generated/deployments/alpha/river in this PR ? This is needed for packages preinstall hook to succeed on a build.

Comment (giuseppecrj) [reply contracts/makefile:130]: should be up now.

**Review by mechanical-turk — APPROVED** [review]
🙌

**Review by texuf — APPROVED** [review]

**Review by jterzis — APPROVED** [review]

File: packages/generated/deployments/alpha/river/addresses/riverRegistry.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"address":"0xD85bB9e2CEcdA2b8Bf39aBcb514C222046029951"}
```
Comment (mechanical-turk) [inline packages/generated/deployments/alpha/river/addresses/riverRegistry.json:1]: @giuseppecrj why does this contract address show up as "multi-init" deployed 6 months ago? 

https://base-sepolia.blockscout.com/address/0xD85bB9e2CEcdA2b8Bf39aBcb514C222046029951

**Pull request merged**